app/controllers/api/account.php

```diff
@@ -2877,7 +2878,7 @@
     ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
     ->label('sdk.response.model', Response::MODEL_TOKEN)
     ->label('abuse-limit', 10)
-    ->label('abuse-key', 'userId:{userId}')
+    ->label('abuse-key', ['url:{url},userId:{userId}', 'ip:{ip}'])
```

Comment:
```suggestion
    ->label('abuse-key', ['url:{url},userId:{userId}', 'ip:{ip}'])
```

don't we need the URL in the IP based abuse key as well ?
Else requests to other endpoints using the same IP may also be blocked 🤔